config.js

```diff
@@ -758,9 +758,9 @@ config.NSFS_GLACIER_MIGRATE_INTERVAL = 15 * 60 * 1000;
 // of `manage_nsfs glacier restore`
 config.NSFS_GLACIER_RESTORE_INTERVAL = 15 * 60 * 1000;
 
-// NSFS_GLACIER_EXPIRY_INTERVAL indicates the interval between runs
-// of `manage_nsfs glacier expiry`
-config.NSFS_GLACIER_EXPIRY_INTERVAL = 12 * 60 * 60 * 1000;
+// NSFS_GLACIER_EXPIRY_RUN_TIME must be of the format hh:mm which specifies
+// when NooBaa should allow running glacier expiry process
+config.NSFS_GLACIER_EXPIRY_RUN_TIME = '03:00';
```

Comment:
worth noting here that this is also in the timezone option as set by NSFS_GLACIER_EXPIRY_TZ and NSFS_GLACIER_EXPIRY_TIME_OF_DAY